src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -40,6 +41,17 @@ public void testSampleSelection() throws IOException {
         spec.executeTest("testSampleSelection--" + testFile, this);
     }
 
+
+    @Test
+    public void testBrokenThing() throws IOException {
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addVCF(new File("denovo_2-5pct_1-filtered.vcf"));
+        args.addReference(new File("/Users/louisb/data/Homo_sapiens_assembly19.fasta"));
+        args.addOutput(new File("out.vcf"));
+
+        runCommandLine(args);
+    }
```

Comment:
Assuming inclusion of this is an oversight ?

Reply:
Ack, yes, that is definitely an oversight...